lib/util.js

```diff
@@ -277,14 +277,17 @@ const util = {
       const androidNtpTilesResDest = path.join(config.srcDir, 'components', 'ntp_tiles', 'resources')
       const androidResTemplateSource = path.join(config.projects['brave-core'].dir, 'android', 'java', 'res_template')
       const androidResTemplateDest = path.join(config.srcDir, 'chrome', 'android', 'java', 'res_template')
+      const androidSource = path.join(config.projects['brave-core'].dir, 'chromium_src', 'chrome', 'android')
```

Comment:
Maybe call it `androidJavaSource`/`androidJavaDest`?
Other looks good

Reply:
I'm confused about what you're trying to do here, why would we copy java files into the chrome directory?

Reply:
I understand we want to overwrite Chromium's `*.java` file completely with Brave's to avoid gn-file 
 patching, like we do for `*.cc` and `chromium_src`

Reply:
we don't do that for .cc files. We have a wrapper script that uses the chromium src in place of the original source and still allows us to reference the original. We do not copy the files in the chrome source directory

Reply:
then just to completely replace Chromium's *.java files Brave's *.java files